src/harlequin/components/adapter_details.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
```

Comment:
Needed for 3.9 compatibility:

```suggestion
from __future__ import annotations

from pathlib import Path
```